server/tests/model/test_family.py

```diff
@@ -0,0 +1,54 @@
+"""Test the family model."""
+
+from typing import Generator
+from unittest import mock
+
+import pytest
+from mongoengine import connect, disconnect
+from src.model.family import Family
+from src.utils.error import ErrorCode
+
+
+@pytest.fixture
```

Comment:
Not sure if possible, but can we import from a different file? This method seems to be repeated in multiple files.

Reply:
Worth looking into, but yeah not sure if that would just create one global mock DB instead of individual ones for each test file.

Reply:
Pretty sure your fixture is creating a mock db for every test. This behavior can be changed tho - https://docs.pytest.org/en/latest/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

Reply:
Pretty sure your fixture is creating a mock db for every test. This behavior can be changed tho - https://docs.pytest.org/en/latest/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session